jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/SpringBootUtilTest.java

```diff
@@ -74,4 +81,42 @@ public void testMultipleProfilesParsing() {
         assertNull(props.get("spring.application.name"));
     }
 
+    @Test
+    public void getSpringBootPluginConfiguration_whenSpringBootMavenPluginPresent_thenReturnsPluginConfiguration() {
+        // Given
+        JavaProject javaProject = JavaProject.builder()
+            .plugin(Plugin.builder()
+                .groupId(SPRING_BOOT_GROUP_ID)
+                .artifactId(SPRING_BOOT_MAVEN_PLUGIN_ARTIFACT_ID)
```

Comment:
Please, use a string here, if there's an error/typo in the constant it won't be revealed